### PR TITLE
Add advanced options block to cloud remote connection

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change
CleanShot 2024-02-28 at 14 31 50

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -3508,7 +3508,11 @@
               "instance_will_be_available": "Your instance will be available at your",
               "link_learn_how_it_works": "Learn how it works",
               "nabu_casa_url": "Nabu Casa URL",
-              "certificate_info": "Certificate info"
+              "external_activation": "Allow external activation",
```
Comment (c0ffeeca7) [inline src/translations/en.json:3511]:
```suggestion
              "external_activation": "Allow external activation of remote control",
```

File: src/panels/config/cloud/account/cloud-remote-pref.ts [hunk 2]
```diff
@@ -125,22 +128,55 @@ export class CloudRemotePref extends LitElement {
...
-            ></ha-switch>
-          </ha-formfield>
-        </div>
-        <div class="card-actions">
-          <mwc-button @click=${this._openCertInfo}>
-            ${this.hass.localize(
-              "ui.panel.config.cloud.account.remote.certificate_info"
-            )}
-          </mwc-button>
+          <ha-expansion-panel outlined .header=${"Advanced options"}>
```
Comment (piitaya) [inline src/panels/config/cloud/account/cloud-remote-pref.ts:134]: We should translate the header.

**Review by piitaya — CHANGES_REQUESTED** [review]

File: src/translations/en.json [hunk 3]
```diff
@@ -3508,7 +3508,11 @@
...
+              "external_activation_secondary": "Should you be able to turn on remote control from your Nabu Casa account page.",
```
Comment (c0ffeeca7) [inline src/translations/en.json:3512]:
```suggestion
              "external_activation_secondary": "Allows you to turn on remote control from your Nabu Casa account page, even if you're outside your local network.",
```
would it be worth mentioning that this works if I'm outside my network? I guess that's the whole point of it, but I'm not sure how obvious it is.

Comment (bramkragten) [reply src/translations/en.json:3512]: Went for your suggestion, looks good

**Review by piitaya — APPROVED** [review]

**Pull request merged**